lib/tasks/test.js

```diff
@@ -264,5 +271,45 @@ module.exports.browserTest = function(gulp, config) {
 	});
 };
 
+module.exports.pa11yTest = function(gulp, config) {
+	const test = pa11y({
+		log: {
+			debug: console.log.bind(console),
+			error: console.error.bind(console),
+			info: console.info.bind(console)
+		},
+		ignore: ['WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID']
```

Comment:
Might be worth commenting what these code reference to, or where to find that information